app/pb/communities.proto

```diff
@@ -0,0 +1,105 @@
+syntax = "proto3";
+
+package org.couchers.api.communities;
+
+import "google/protobuf/timestamp.proto";
+
+service Communities {
+  // This is a secure service: a user needs to be authenticated and not jailed to call functions here, refer to auth.proto and jail.proto
+
+  // Discussion threads are organized in three levels:
+  // thread1
+  // +- comment1
+  // |  +- reply1
+  // |  +- reply2
+  // +- comment2
+  //    +- reply3
+  //    +- reply4
+  // thread2
+  // +- comment3
+  // |  +- reply5
+  // |  +- reply6
+  // +- comment4
+  //    +- reply7
+  //    +- reply8
+
+  // Get non-deleted comments belonging to a thread
+  rpc GetComments(GetCommentsReq) returns (GetCommentsRes);
```

Comment:
Could you please specify in a comment the ordering and logic of returned comments. It's not so clear at the moment. Something like

```py
// Returns the last `number` non-deleted comments before `posted_before_comment_id`
```

Am I understanding this correctly? So the client would normally pull, say the last 3 comments?

Reply:
I documented this in `GetCommentsReq` but maybe it is better to move all the documentation to the this rpc definition. I think a typical client will request 100 comments at a time.